assignment-2/src/view/dumb/MyVoteSystem.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {FaChevronDown, FaChevronUp} from "react-icons/fa";
+
+const MyVoteSystem = ({questionAnswer, onVote, sOUserName}) => (
+    <div style={{"-webkit-user-select": "none"}}>
+        <h1 className="alignCenter vote"
+            onClick={() => {
```

Comment:
There is a better way to write this: `onClick={questionAnswer.userName !== sOUserName ? onVote(...) : undefined}`